client/src/components/Home/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import NavBar from "../Navbar";
+import "./style.css";
+import Category from "../Category";
+class Home extends React.Component {
+  goToShop = () => {
+    const { history } = this.props;
+    history.push("/shop");
```

Comment:
The category end point should contain the category id like `/shop/:category_id`

Reply:
i add end point with id !